.changeset/long-rabbits-roll.md

```diff
@@ -1,27 +1,35 @@
 ---
-"@logto/cli": minor
+"@logto/tunnel": major
```

Comment:
should it be minor?

Reply:
Personally I'd like to see a `v1.0.0` upon publish.